src/ui/screens/onboarding/Page.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { FragmentSafeAreaViewForTabBar } from '../../../ui/misc/FragmentSafeAreaView';
+import { ReactNode } from 'react-redux';
+import {
+    View,
+    TouchableWithoutFeedback,
+    StyleSheet,
+} from 'react-native';
+import {
+    DefaultTabBarHeight,
+    Colors,
+} from '../../../styles';
+import { MediumText } from '../../../ui/misc/text';
+import { ButtonProps } from '../../../components/NavigationHeader';
+
+interface Props {
+    backgroundColor: string;
+    safeAreaTopBackgroundColor?: string;
+    children: ReactNode | ReactNode[];
+    leftButton?: ButtonProps;
+    rightButton?: ButtonProps;
+}
+
+export const Page = (props: Props) => {
+    return (
+        <FragmentSafeAreaViewForTabBar
```

Comment:
As there is no tab bar during the onboarding, shouldn't we use the `FragmentSafeAreaViewWithoutTabBar`?

Reply:
there is, it's just not the react-navigation tabbar

Reply:
I don't really understand then

I thought the distinction between the two FragmentSafeAreaView is because the react-navigation tabbar is positioned absolutely from the bottom of the page and it's transparent, so the bottom unsafe area would have to styled differently and a padding should be added.

Also if you use the `FragmentSafeAreaViewWithoutTabBar` here there is no visible difference.

Reply:
You also need it because of the different color for the top and bottom part of the safearea. The naming is probably still wrong.